shell/browser/ui/electron_desktop_window_tree_host_linux.cc

```diff
@@ -241,4 +242,24 @@ void ElectronDesktopWindowTreeHostLinux::UpdateFrameHints() {
     SizeConstraintsChanged();
   }
 }
+
+void ElectronDesktopWindowTreeHostLinux::DispatchEvent(ui::Event* event) {
+  if (event->IsMouseEvent()) {
+    auto* mouse_event = static_cast<ui::MouseEvent*>(event);
+    bool should_disable_drag =
+        mouse_event->IsRightMouseButton() ||
+        (mouse_event->IsLeftMouseButton() && mouse_event->IsControlDown());
+    if (should_disable_drag)
+      electron::api::WebContents::SetDisableDraggableRegions(true);
+
+    views::DesktopWindowTreeHostLinux::DispatchEvent(event);
+
+    if (should_disable_drag)
+      electron::api::WebContents::SetDisableDraggableRegions(false);
```

Comment:
can this be combined ? would `should_disable_drag` change between the calls ?

```suggestion
    if (should_disable_drag) {
      electron::api::WebContents::SetDisableDraggableRegions(true);
      views::DesktopWindowTreeHostLinux::DispatchEvent(event);
      electron::api::WebContents::SetDisableDraggableRegions(false);
      return;
   }
```